cmake/sysbuild/suit.cmake

```diff
@@ -176,13 +176,19 @@ function(suit_create_package)
   foreach(image ${IMAGES})
     sysbuild_get(BINARY_DIR IMAGE ${image} VAR APPLICATION_BINARY_DIR CACHE)
     sysbuild_get(BINARY_FILE IMAGE ${image} VAR CONFIG_KERNEL_BIN_NAME KCONFIG)
+    sysbuild_get(target IMAGE ${image} VAR CONFIG_SUIT_ENVELOPE_TARGET KCONFIG)
```

Comment:
'target' is for example "radio" and "application" (instead of "image": where we have for example "hci_ipc"). Manifests use "radio" so if we want this tp be accessible to all manifests we need to put this line here.